README.md

```diff
@@ -1 +1,11 @@
-# pandemic-game
\ No newline at end of file
+# pandemic-game
+
+## Getting started
+
+### Running in development mode
+Run: `npm run start:dev`
```

Comment:
Could you provide some instructions for how someone can get their `npm` environment set up overall?

On Ubuntu, I first ran `sudo apt install npm webpack`.
Then I tried running `npm run start:dev` as recommended. I got the following output:

```
(base) ubuntu@ip-172-31-26-119:~/pandemic-game-2$ npm run start:dev                                                                                                                                       
                                                                                                     
> pandemic-game@1.0.0 start:dev /home/ubuntu/pandemic-game-2
> webpack serve --config webpack.config.js                                                           
                                                                                                     
The command moved into a separate package: @webpack-cli/serve
Would you like to install serve? (That will run npm install -D @webpack-cli/serve) (yes/NO) : yes
                                                                                                                                                                                                          
> core-js@3.7.0 postinstall /home/ubuntu/pandemic-game-2/node_modules/@babel/node/node_modules/core-js
> node -e "try{require('./postinstall')}catch(e){}"
                                                                                                     
Thank you for using core-js ( https://github.com/zloirock/core-js ) for polyfilling JavaScript standard library!
                                                                                                                                                                                                          
The project needs your help! Please consider supporting of core-js on Open Collective or Patreon: 
> https://opencollective.com/core-js                                                                                                                                                                      
> https://www.patreon.com/zloirock                                                                                                                                                                        
                                                  
Also, the author of core-js ( https://github.com/zloirock ) is looking for a good job -)
                                                  
                                                                                                     
> core-js@2.6.11 postinstall /home/ubuntu/pandemic-game-2/node_modules/core-js
> node -e "try{require('./postinstall')}catch(e){}"                                                  
                                                                                                     
                                                                                                                                                                                                          
> core-js-pure@3.7.0 postinstall /home/ubuntu/pandemic-game-2/node_modules/core-js-pure
> node -e "try{require('./postinstall')}catch(e){}"
                                                  
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.13 (node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.13: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@2.1.3 (node_modules/chokidar/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.1.3: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
                                                  
+ @webpack-cli/serve@1.1.0
added 874 packages from 427 contributors and audited 880 packages in 22.842s
                                                  
62 packages are looking for funding                                                                                                                                                                       
  run `npm fund` for details                                                                         

found 0 vulnerabilities         
                                                  
[webpack-cli] Unknown argument: /usr/bin/node                                                        
[webpack-cli] Unknown argument: /usr/bin/webpack                                                     
npm ERR! code ELIFECYCLE                                                                             
npm ERR! errno 2
npm ERR! pandemic-game@1.0.0 start:dev: `webpack serve --config webpack.config.js`
npm ERR! Exit status 2
npm ERR! 
npm ERR! Failed at the pandemic-game@1.0.0 start:dev script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/ubuntu/.npm/_logs/2020-11-15T20_09_54_657Z-debug.log
```


Running `npm run start:dev` again, I now get the following:

```
(base) ubuntu@ip-172-31-26-119:~/pandemic-game-2$ npm run start:dev                                   
                                                                                                     
> pandemic-game@1.0.0 start:dev /home/ubuntu/pandemic-game-2                                                                                                                                              
> webpack serve --config webpack.config.js                                                                                                                                                                
                                                                                                                                                                                                          
ℹ ｢wds｣: Project is running at http://localhost:9000/
ℹ ｢wds｣: webpack output is served from undefined                                                     
ℹ ｢wds｣: Content not from webpack is served from /home/ubuntu/pandemic-game-2/dist                   
⚠ ｢wdm｣: WARNING in asset size limit: The following asset(s) exceed the recommended size limit (244 KiB).
This can impact web performance.                                                                                                                                                                          
Assets:                                                                                              
  bundle.js (289 KiB)
                                                  
WARNING in entrypoint size limit: The following entrypoint(s) combined asset size exceeds the recommended limit (244 KiB). This can impact web performance.
Entrypoints:                                                                                         
  main (289 KiB)                                                                                     
      bundle.js                                                                                      
                                                  
                                                                                                     
WARNING in webpack performance recommendations:   
You can limit the size of your bundles by using import() or require.ensure to lazy load some parts of your application.
For more info visit https://webpack.js.org/guides/code-splitting/                                    
                                                                                                     
webpack compiled with 3 warnings                                                                     
ℹ ｢wdm｣: Compiled with warnings.
```

This looks like it might work, but then when I visit http://3.134.253.143:9000/ (the IP address of my AWS instance), I get "This site can’t be reached" (ERR_CONNECTION_REFUSED). I'm pretty sure it is not a port-opening issue, as I have allowed all ports to be open on this AWS instance (and indeed, http://3.134.253.143:8000/ is still up and running, serving the plain HTML version).

I think a bit more guidance could help new people get up and running faster. Though people experienced with `npm` will have no problem, I'm sure.